app/utils/RepaymentAmountSummaryAnswersHelper.scala

```diff
@@ -123,14 +123,14 @@ class RepaymentAmountSummaryAnswersHelper(userAnswers: UserAnswers)(implicit mes
   private def getAnswerSection(dutyType: String, dutyPaid: Double, dutyDue: Double): AnswerSection = {
     val answerSection: AnswerSection = dutyType match {
       case "repaymentAmountSummary.total" =>
-        AnswerSection(Some(dutyType), Seq(displayDuty("0", dutyPaid, "repaymentAmountSummary.total.amount").get))
+        AnswerSection(Some(dutyType), Seq(displayDuty("0", dutyPaid, "repaymentAmountSummary.total.amount")))
```

Comment:
Found these problems when doing some "extreme" testing of missing answers (i.e. all answers missing).  Better to render an empty row than to blow up I think.